src/components/tank-monitoring.vue

```diff
@@ -57,7 +57,7 @@ import {
   Version,
   BrewhopsCookie,
   BatchUpdateOrCreate
-} from '../types';
+} from '../types/index';
```

Comment:
Can we be consistent with paths, relative vs absolute, or if you have a reason for this, will you explain it.

Reply:
Of course. Keep in mind I did a lot of refactoring so there may be some imports that I missed. I'm going to fix them.

Reply:
I believe I've gotten them all.